src/test/regress/expected/multi_cluster_management.out

```diff
@@ -676,8 +676,122 @@ SELECT master_update_node(:worker_1_node, 'localhost', :worker_1_port);
 (1 row)
 
 SELECT * FROM pg_dist_node WHERE nodeid = :worker_1_node;
- nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole | nodecluster 
---------+---------+-----------+----------+----------+-------------+----------+----------+-------------
-     16 |      14 | localhost |    57637 | default  | f           | t        | primary  | default
+ nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole | nodecluster | isdatanode 
+--------+---------+-----------+----------+----------+-------------+----------+----------+-------------+------------
+     16 |      14 | localhost |    57637 | default  | f           | t        | primary  | default     | true
 (1 row)
 
+SET citus.shard_replication_factor TO 1;
+CREATE TABLE test_dist (x int, y int);
+SELECT create_distributed_table('test_dist', 'x');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- testing behaviour when setting isdatanode to 'marked for removal'
+UPDATE pg_dist_node SET isdatanode = 'marked for removal' WHERE nodeport = :worker_2_port;
+CREATE TABLE test_dist_colocated (x int, y int);
+CREATE TABLE test_dist_non_colocated (x int, y int);
+CREATE TABLE test_dist_colocated_with_non_colocated (x int, y int);
+CREATE TABLE test_ref (a int, b int);
+SELECT create_distributed_table('test_dist_colocated', 'x');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_distributed_table('test_dist_non_colocated', 'x', colocate_with => 'none');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_distributed_table('test_dist_colocated_with_non_colocated', 'x', colocate_with => 'test_dist_non_colocated');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_reference_table('test_ref');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+-- colocated tables should still be placed on 'marked for removal' nodes for safety
+SELECT nodeport, count(*)
+FROM pg_dist_shard JOIN pg_dist_shard_placement USING (shardid)
+WHERE logicalrelid = 'test_dist_colocated'::regclass GROUP BY nodeport;
+ nodeport | count 
+----------+-------
+    57637 |     2
+    57638 |     2
+(2 rows)
+
+-- non colocated tables should still be placed on 'marked for removal' nodes for safety
```

Comment:
comment seems to be copied from above.
I would expect a non colocated table to _not_ be placed on `marked for removal` nodes.